### ISSUE TITLE
Combobox support

### ISSUE THREAD

**Issue by froggushka** [flyingxu/Bootstrap.WPF#5]
Hello,
I have been looking at the wpf library for a bit now and I got to say it looks very nice. I don't have much experience with UI themes but even I can make my screen look more or less nice with it. I did note one UI components that seems to stick out and I was just wondering if you have any intent in the future to also provide a theme for Combobox.

Thank you for you time and keep up the good work, Chris

**Comment by flyingxu** [comment]
Yes, sure. I think a bootstrap style for WPF should be a good option for those who want a nice UI style but don't want to go too far, just like me. But too sad that the original author moved his interest away from WPF so I must be on my own and create this fork.
I'm maintaining this fork as a hobby project and also I use it myself in my paid job. I would like to add more controls, for example, ComboBox, but please give me some time. 
At the same time, any help is welcomed.
